pkg/pipelines/create_cluster.go

```diff
@@ -90,6 +90,7 @@ func NewCreateClusterPipeline(runtime *common.KubeRuntime) error {
 		&kubernetes.ConfigureKubernetesModule{},
 		&filesystem.ChownModule{},
 		&certs.AutoRenewCertsModule{Skip: !runtime.Cluster.Kubernetes.EnableAutoRenewCerts()},
+		&kubernetes.SecurityEnhancementModule{Skip: runtime.Arg.SecurityEnhancement},
```

Comment:
if the value of `runtime.Arg.SecurityEnhancement` is true, this module will be skipped.